<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Проект</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="copySubMenu">
     <property name="title">
      <string>Скопировать</string>
     </property>
     <addaction name="copyDirAct"/>
     <addaction name="copyFileAct"/>
    </widget>
    <widget class="QMenu" name="createSubMenu">
     <property name="title">
      <string>Создать</string>
     </property>
     <addaction name="createDirAct"/>
     <addaction name="createFileAct"/>
    </widget>
    <addaction name="createSubMenu"/>
    <addaction name="copySubMenu"/>
    <addaction name="separator"/>
    <addaction name="saveProjectAct"/>
    <addaction name="separator"/>
    <addaction name="exitMenuAct"/>
    <addaction name="exitAct"/>
   </widget>
   <widget class="QMenu" name="ViewMenu">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="projectInformationAct"/>
    <addaction name="fileTreeAct"/>
    <addaction name="kanbanAct"/>
   </widget>
   <widget class="QMenu" name="runMenu">
    <property name="title">
     <string>Управление</string>
    </property>
    <addaction name="listOfTasksAct"/>
    <addaction name="addTaskAct"/>
    <addaction name="separator"/>
    <addaction name="setDeadlineAct"/>
    <addaction name="editDeadlineAct"/>
    <addaction name="separator"/>
    <addaction name="delProjectAct"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="ViewMenu"/>
   <addaction name="runMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_3">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="saveProjectAct">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="projectInformationAct">
   <property name="text">
    <string>Информация о проекте</string>
   </property>
  </action>
  <action name="tasksViewAct">
   <property name="text">
    <string>Задачи</string>
   </property>
  </action>
  <action name="fileTreeAct">
   <property name="text">
    <string>Ветка файлов</string>
   </property>
  </action>
  <action name="addTaskAct">
   <property name="text">
    <string>Добавить задачу</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Удалить задачу</string>
   </property>
  </action>
  <action name="addSubtaskAct">
   <property name="text">
    <string>Добавить подзадачу</string>
   </property>
  </action>
  <action name="setDeadlineAct">
   <property name="text">
    <string>Установить дедлайн</string>
   </property>
  </action>
  <action name="editDeadlineAct">
   <property name="text">
    <string>Изменить дедлайн</string>
   </property>
  </action>
  <action name="delProjectAct">
   <property name="text">
    <string>Удалить проект</string>
   </property>
  </action>
  <action name="exitAct">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="listOfTasksAct">
   <property name="text">
    <string>Список задач</string>
   </property>
  </action>
  <action name="kanbanAct">
   <property name="text">
    <string>Кан-бан</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Выйти в главное меню</string>
   </property>
  </action>
  <action name="copyDirAct">
   <property name="text">
    <string>Директорию</string>
   </property>
  </action>
  <action name="copyFileAct">
   <property name="text">
    <string>Файл</string>
   </property>
  </action>
  <action name="createDirAct">
   <property name="text">
    <string>Директорию</string>
   </property>
  </action>
  <action name="createFileAct">
   <property name="text">
    <string>Файл</string>
   </property>
  </action>
  <action name="exitMenuAct">
   <property name="text">
    <string>Выход в меню</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
